ddocs/medic-db/medic-client/validate_doc_update.js

```diff
@@ -13,11 +13,42 @@ function(newDoc, oldDoc, userCtx) {
     throw({ forbidden: msg });
   };
 
+  var hasRole = function(roles, role) {
```

Comment:
Copied from the server validate_doc_update function. We don't yet have a sensible way to include functions like this.